src/app/js/components/files/context-menu.js

```diff
@@ -13,13 +13,19 @@ class FilesContextMenu extends Component {
         <MenuItem onClick={onMoveDir}>
           Rename
         </MenuItem>
+        {selectedFiles.length === 1 && onCopyHash ? (
+          <MenuItem onClick={onCopyHash}>
+            Copy hash to clipboard
```

Comment:
I would change to 'Copy Hash' only. I think it's better since the 'clipboard' part is implied. 😄